src/ngraph/op/fused/space_to_depth.cpp

```diff
@@ -41,6 +42,13 @@ op::SpaceToDepth::SpaceToDepth(const Output<Node>& data, const std::string& mode
 {
 }
 
+bool ngraph::op::v0::SpaceToDepth::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("blocksize", m_blocksize);
```

Comment:
Incorrect name of attribute! Please change it to `block_size`

Reply:
Done